### PR TITLE
Fix #623: Shift StoryTesting files to testing package

### LINKED ISSUE #623: Shift StoryTesting files to testing package
`StoryFragmentTestActivity` and `StoryFragmentTestActivityPresenter` files are currently in **app** module -> **story** -> **testing** package. 

Shift this to **app** module -> **testing** package. 

Also, where these files are getting used like `story_fragment_test_activity.xml`, fix those usage too.

### PR DESCRIPTION
**Pull request by NullByte08**
## Explanation

Moved files StoryFragmentTestActivity and StoryFragmentTestActivityPresenter  to **app** module-> **testing** package. And fixed the references and usages of the files.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -157,4 +162,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```
Comment (rt4914) [inline .idea/codeStyles/Project.xml]: We don't need any change to `.idea` file, this must have been done automatically from android studio please do not add this file to your commit. 
Note: Don't delete this file, we need this file we just don't need the changes this file contains.

File: app/src/main/java/org/oppia/app/story/testing/StoryFragmentTestActivityPresenter.kt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package org.oppia.app.story.testing
+package org.oppia.app.testing
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/story/testing/StoryFragmentTestActivityPresenter.kt:1]: This is incorrect. You have changed the package name but the file is still inside `app/story/testing`, instead it should be shifted to `app/testing`

File: app/src/main/java/org/oppia/app/story/testing/StoryFragmentTestActivity.kt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package org.oppia.app.story.testing
+package org.oppia.app.testing
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/story/testing/StoryFragmentTestActivity.kt:1]: This is incorrect. You have changed the package name but the file is still inside `app/story/testing`, instead it should be shifted to `app/testing`

**Review by rt4914 — CHANGES_REQUESTED** [review]
Changes suggested.

**Comment by rt4914** [comment]
@NullByte08 Your PR title should be `Fix #623: Shift StoryTesting files to testing package`. 
You are missing `:`

**Comment by NullByte08** [comment]
@rt4914 Please review.

**Review by rt4914 — APPROVED** [review]
LGTM (Look's Good To Me). Thanks. You can work on the other issue assigned to you.

**Pull request merged**